### PR TITLE
Link to the discourse group within tutorials

### PR DESCRIPTION
**Pull request by SimonBiggs**
Change to linking to the Discourse Group

### CODE REVIEW

File: lib/pymedphys/docs/tutes/index.rst [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-`NSA Python course`_ written by the creator of Python.
+The goal for these tutorials is to have a suite of Jupyter Notebooks that can
+be interactively run within the documentation to help Medical Physicists who
+have no Python experience begin to have success using the tools within the
+Medical Physics ecosystem. To build a suite of tutorials like this is a big
+task, and most of the developers of PyMedPhys are focused on developing
+PyMedPhys itself. Are you reading this and thinking that you might be able to
+manage and facilitate the creation of a suite of Notebooks like this? You
+certainly don't have to create them all by yourself, instead, just take the
+task by the horns design the overall tutorial flow/course, and find people who
```
Comment (SimonBiggs) [inline lib/pymedphys/docs/tutes/index.rst:18]:
```suggestion
task by the horns, design the overall tutorial flow/course, and find people who
```

File: lib/pymedphys/docs/tutes/index.rst [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+can build each component.
 
-.. _`NSA Python course`: https://ia802805.us.archive.org/7/items/comp3321/comp3321_text.pdf
+Please reach out by sending me (@SimonBiggs) a private message on the Discourse
```
Comment (SimonBiggs) [inline lib/pymedphys/docs/tutes/index.rst:21]:
```suggestion
If you would like to consider taking this on
please reach out by sending me (@SimonBiggs) a private message on the Discourse
```

File: lib/pymedphys/docs/README.rst [hunk 3]
```diff
@@ -92,7 +92,7 @@ For Python Users
   - Start here if you are fluent in Python and just want a quick instruction on how
     to get started.
 
-- `Tutorials`_ **[Don't yet exist]**
+- `Tutorials`_
```
Comment (SimonBiggs) [inline lib/pymedphys/docs/README.rst:95]: This is a little cheeky, yes. But... I want the notice linked to, to be able to be read...

File: README.rst [hunk 4]
```diff
@@ -88,7 +88,8 @@ Documentation
 For App Users
 .............
 
-App documentation doesn't yet exist. Watch this space.
+Documentation of the PyMedPhys application's graphical user interface doesn't
+yet exist. Watch this space.
```
Comment (SimonBiggs) [inline README.rst:92]: While I was in the area I saw that this read like "no documentation at all" existed. Clarified that it meant documentation of the GUI.

**Review by sjswerdloff — APPROVED** [review]
I had not noticed serviceplans before.
Separate discussion regarding the choice of "service" for the name of the module/package.

**Comment by SimonBiggs** [comment]
> I had not noticed serviceplans before.
> Separate discussion regarding the choice of "service" for the name of the module/package.

Yup, let's discuss in an issue?

**Pull request merged**